tox.ini

```diff
@@ -35,7 +35,9 @@ commands =
 commands =
     mkdir -p system_test_results/
     pytest \
-    --junitxml=system_test_results//unit_xunit.xml \
+    -o log-cli=true \
+    --log-cli-level INFO \
```

Comment:
Would it be better to have DEBUG level text outputs?

Reply:
I think this would be very verbose. Main point was to have some output so that travis does not think the test is dead.

Reply:
Sounds good then. Let's go ahead with this.